### ISSUE TITLE
interface with mashr

### ISSUE THREAD

**Issue by stephens999** [stephenslab/udr#22]
we should document how to use this with mashr, probably in the mashr vignette?
It may require writing a function or functions to produce mashr objects (priors?) in the udr package?

**Comment by pcarbo** [comment]
Yes, that's a good idea, but perhaps it makes sense to tackle Issue #21 first, and then illustrate the (more user-friendly) interface in the vignette.

**Comment by stephens999** [comment]
related to https://github.com/stephenslab/mashr/issues/132 
- does udr handle missing data?
- Can we fit both the EE and EZ model with udr?

**Comment by yunqiyang0215** [comment]
@stephens999 udr handles missing data using mashr function. The input to [ud_init()](https://github.com/stephenslab/udr/blob/master/R/ud_init.R) can be a mashr object, and handling missingness part is using mashr. 

We can fit both EE and EZ model with udr.

**Comment by stephens999** [comment]
In mashr, V is the correlation matrix, so the covariance is SVS.
Is it the same in udr software?
(I'm asking because in the udr paper we use V for the covariance)
.

**Comment by yunqiyang0215** [comment]
I think V is for covariance for udr.

**Comment by stephens999** [comment]
i think we should think a bit about this.
If we keep that different convention in the two packages
then we should probably not mix the mash data object with the udr V as inputs to the same function.

For example, currently we allow both a mash_data object and V to be provided to udr_init, but 
then it's confusing: not clear if the V refers to the udr V or the mashr V.
Maybe it would be enough to not allow the user to specify both? 

Or maybe it would be better to be consistent about the use of V in the two packages?

**Comment by yunqiyang0215** [comment]
Agree... Would be better to be consistent about the notation V. The only concern is if we change udr software, the notation will be different from what we wrote in paper.

**Comment by stephens999** [comment]
it turns out the same is true for mashr: we use V for variance in the paper, but for correlation in the software

**Comment by stephens999** [comment]
@pcarbo do you have thoughts on how to proceed?

**Comment by pcarbo** [comment]
We should do our best to make sure the` udr` interface is consistent with the existing `mashr` interface. (This doesn't necessarily mean we need to change the variable names in the underlying implementation. But the input arguments in the exported functions should be consistent.)

**Comment by stephens999** [comment]
the mashr interface is built around the notion of a "data" object, which contains Bhat, Shat (optional; default is 1) and a correlation matrix (or list of correlation matrices) V. 
Do we want to build udr around the same type of object?

**Comment by pcarbo** [comment]
Definitely!
